notebooks/274-Real-ESRGAN-image-restoration/274-Real-ESRGAN-image-restoration.ipynb

```diff
@@ -0,0 +1,740 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    orig_img = mpimg.imread("../data/image/lr_face.png")
</pre><p>what is the source of input image? Can we use one of provided in this our repo instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/939/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>